eng/pipelines/libraries/run-test-job.yml

```diff
@@ -77,7 +77,7 @@ jobs:
                     $(_buildArguments)
```

Comment:
Is runtime.depproj invoked as part of the pretest subset?

Reply:
`runtime.depproj` is needed to run against the actual intended runtime. The runtime that we're getting via the build-test-job will always be a x64 release runtime because we don't build arch specific tests. We need to override it to whatever OS-arch-configuration we want to run, before actually running the tests.

Reply:
right, I forgot to sequence it into pretest LOL. Thanks